requirements.txt

```diff
@@ -27,3 +27,4 @@ pyparsing>=2.2.0
 lz4>=0.9.0
 tooz>=1.38
 cachetools
+python-qpid-proton>=0.17.0
```

Comment:
We're not going to force everyone to have qpid-proton here. Add an extra in `setup.cfg` and add it to the `doc` too.

Reply:
Will take care of this.